includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -237,6 +283,8 @@ public function prepare_items() {
 		}
 
 		$per_page = $this->get_items_per_page( 'sensei_comments_per_page' );
+
+		/* Filter documented in includes/class-sensei-grading-main.php */
```

Comment:
Maybe we should do the same for the other occurrences of this filter and keep only 1 documented? So we avoid having duplicated documentation. I didn't test what would work with the generated documentation to see if it would get only one or if it would be duplicated.
WDYT?

Reply:
Thanks for an idea to build docs 😄 
I found the opposite approach beneficial!
So I'm going to replace comments like this with a hook doc comment.

Here is why:
![CleanShot 2023-10-10 at 14 33 45@2x](https://github.com/Automattic/sensei/assets/329356/21781ad9-78cd-4d01-8a34-7207422c2492)

In this case, in our docs we can see all places where the filter was applied.

Reply:
Ah, it handles it very well! =)

> I found the opposite approach beneficial!

Agree with this then!

Reply:
I updated the PR replacing comments like "Filter documented {somewhere}" with an actual documentation block.